indico/modules/events/registration/controllers/display.py

```diff
@@ -103,14 +103,16 @@ def _process(self):
                        ~Registration.is_deleted,
                        _join=Registration.registration_form)
                  .order_by(db.func.lower(Registration.last_name), db.func.lower(Registration.first_name)))
-        registrations = [(reg.get_full_name(), reg.get_personal_data().get('affiliation'))
+        registrations = [(reg.get_full_name(), reg.get_personal_data().get('title'),
```

Comment:
Do people really care about the title here? 

Anyway, now that we are calling `get_personal_data` more than once I'd use a helper function to build the tuple to avoid calling it multiple times.

```
def _get_registration_info(registration):
    personal_data = registration.get_personal_data()
    return registration.get_full_name(), personal_data.get('title'), personal_data.get('affiliation')

 registrations = map(_get_registration_info, query)
```